tests/spinup_tests.rs

```diff
@@ -51,6 +51,22 @@ mod spinup_tests {
         testcases::all::http_js_works(CONTROLLER).await
     }
 
+    #[tokio::test]
+    async fn http_python_works() {
+        testcases::all::http_python_works(CONTROLLER).await
+    }
+
+    #[tokio::test]
+    #[ignore] // https://github.com/fermyon/spin/issues/1210
```

Comment:
I _think_ it should function if the '/index.php' path is used (https://github.com/fermyon/spin/issues/1210#issuecomment-1451129360)?

Anyways, follow-up material when ready!

Reply:
yes, i want to update the template for php to make it work out of the box without having to have explicit `index.php`. in a follow up PR